assets/src/styles/blocks/Articles.scss

```diff
@@ -27,6 +23,21 @@
     }
   }
 
+  .editing & {
```

Comment:
Maybe it's better to make a separate file for the admin scss, to help keep the frontend bundle size small.

Reply:
There is actually already one, I missed it! Good one 👍